api/capi/include/nntrainer.h

```diff
@@ -215,6 +215,43 @@ int ml_train_model_set_optimizer(ml_train_model_h model,
 int ml_train_model_set_dataset(ml_train_model_h model,
                                ml_train_dataset_h dataset);
 
+/**
+ * @brief Save the model after training
+ * @details Use this function to save the current model
+ *
+ * @since_tizen 6.5
+ * @param[in] model The NNTrainer model handle to save
+ * @param[in] path_prefix  Path prefix to save the file. This function will save
+ * one or number of files using the given prefix. Note that this does not state
```

Comment:
maybe a small example might be good? or too much?

Reply:
I'll think about it, I guess example can come when with some implementation and testing.

Reply:
cool.